src/toil/jobStores/abstractJobStore.py

```diff
@@ -114,6 +114,13 @@ def deleteJobStore(self):
         """
         raise NotImplementedError()
 
+    def getEnv(self):
+        """
+        Returns a list of relevant environment variables so that the leader can iterate over the list
```

Comment:
And don't mention what a potential caller would want to do with the return value. That is their business. Take a step back and ask yourself "Why are we doing this?". The docstring should describe the *purpose* of the method. What is the intent behind it? Why would a subclass want to implement this method?